chezbetty/locale/fa/LC_MESSAGES/betty.po

```diff
@@ -0,0 +1,736 @@
+msgstr "وایسا ببینم‫…‬ این جوری که می‌شه راحت از بتی دزدی کرد؟‫!‬"
+
+#: chezbetty/templates/about.jinja2:63
+msgid "Yes. (But please don't)."
+msgstr "بله، ولی مرامی نکنید‫!‬"
+
+#: chezbetty/templates/about.jinja2:64
+msgid ""
+"The idea is to provide a convenient service to members of the UM CSE "
+"community. Betty makes a small profit off each sale, some of which will be "
+"used to cover the inevitable losses. If too many people take advantage of "
+"Betty's trusting nature, the service will have to shut down :(."
+msgstr ""
+"هدف ما از به راه انداختن این سیستم خدمت به اعضای دانشکده کامپیوتر است. "
+"ما از هر فروش سودی کمی کسب می‌کنیم که بخشی از آن صرف پرداخت ضررهایمان می‌شود. "
+"اگر افراد زیادی از اعتماد بتی سوء استفاده کنند متاسفانه چاره‌ای جز پلمپ "
+"تعاونی نخواهیم داشت."
+
+#: chezbetty/templates/about.jinja2:67
+msgid "Tell me more about my Chez Betty account?"
+msgstr "حساب کاربری من به چه درد می‌خورد؟"
+
+#: chezbetty/templates/about.jinja2:69
+msgid ""
+"Chez Betty sells things nearly at-cost (usually about a 15% markup). Since "
+"no one likes change and Betty is a self-service operation, it's set up so "
+"that each user has an account with Betty. You can put in $20 (or however "
+"much) at a time and use that balance until it runs out."
+msgstr ""
+"بتی اجناس خود را با سود بسیار کم (معمولا حدود ۱۵٪) به فروش می‌رساند. از آنجایی که "
+"بتی یک سرویس خودکار است و امکان بازگرداندن پول خرد را ندارد بقیه پول واریز شده "
+"را در حساب شما برای خرید‌های بعدی نگهداری می‌کند. شما می‌توانید مبلغی (مثلا ۲۰ دلار) "
+"به حساب خود واریز کنید و به تدریج از آن استفاده کنید."
+
+#: chezbetty/templates/about.jinja2:75
+msgid "What if I don't have any money / Can my account balance go negative?"
+msgstr "اگر پول نداشتم چی؟ آیا می‌شود از بتی نسیه گرفت؟"
+
+#: chezbetty/templates/about.jinja2:77
+msgid ""
+"Yes. Chez Betty allows you to spend beyond your immediate means under that "
+"assumption that you'll pay Betty back eventually. The idea is to let you "
+"grab stuff when you need it and keep track of what owe, instead of "
+"incentivizing folks to steal. There is a <a href=\"/shame\">Wall of Shame</"
+"a> for persons who owe Betty more than $5."
+msgstr ""
+"بله. بتی با این پیش‌فرض که شما بدهی خود را در آینده نزدیک پرداخت می‌کنید به شما "
+"نسیه می‌دهد. هدف بتی این است که کاربران بتوانند در صورت همراه نداشتن پول هم از "
+"سرویس بتی بهره‌مند شوند. برای دیدن لیست افرادی که بیشتر از ۵ دلار به بتی بدهکارند "
+"به  <a href=\"/shame\">لیست بدحساب‌ها‪</‬a‪>‬ مراجعه کنید."
+
+#: chezbetty/templates/about.jinja2:85
+msgid "How does Betty set prices?"
+msgstr "بتی چگونه اجناس را قیمت‌گذاری می‌کند؟"
+
+#: chezbetty/templates/about.jinja2:87
+msgid ""
+"Betty sells things just above wholesale, usually a 15% markup. If you're "
+"interested in the actual algorithm, check out Betty's source code at github."
+"com/um-cseg/chez-betty."
+msgstr ""
+ "چز بتی اجناس خود را با سود اندکی ‫(‬معمولا ۱۵٪‫)‬ نسبت به کلی فروشی به فروش می‌رساند‫.‬"
```

Comment:
Github is highlighting this line as a syntax error, I'm guessing it's because of the leading space before the "?